SPECS/ntopng/ntopng.signatures.json

```diff
@@ -1,6 +0,0 @@
-{
- "Signatures": {
```

Comment:
Jon, you might also want to rever the changes in `cgmanifest.json` and the licence map for this package. #608 added `ntopng`, so I used that as a reference of what needs to be reverted.

Reply:
yeah...I realized the same right after I pushed.  How does it look now?

Reply:
Looks good to me!